### ISSUE TITLE
deepin-filemanager doesn't accept native deepin-gtk

### ISSUE THREAD

**Issue by oberon2007** [linuxdeepin/developer-center#239]
I know I have complained about deepin-file-manager's theming before (like many others)... but I need to ask again:
Is it really possible that native **deepin-file-manager** will not even accept the dark variant of native **deepin-gtk-theme**? Does deepin-file-manager seriously only have one single hard-coded style? Seriously? :stuck_out_tongue_winking_eye:

**Comment by hualet** [comment]
It won't of course, but at the moment, the anwser sadly is YES.

**Comment by Acris** [comment]
@hualet How about the file manager's font size? hard coded too? See #236.

**Comment by hualet** [comment]
@dragondjf

**Comment by slinstj** [comment]
No dark theme at all? Is it really? This is the most beauty linux I have used since the 2002 year! I just cant believe the file manager, a so important application in any OS, will continue without a dark theme! Comparing to the Deepin, it is like if the Spider Man would gain a new black clothe but that one would come lacking a sleeve! He would still be the Spider Man, but the respect would not be the same :-1:

**Comment by Skillzore** [comment]
Really missing this dark theme on the file manager as well. Almost enough to want to make me switch desktop environment.... almost. Also, the dark them has a parsing error:
`Gtk-WARNING **: Theme parsing error: gtk.css:3601:22: 'none' is not a valid color name"`

**Comment by oberon2007** [comment]
Deepin-file-manager is still sticking out like a sore thumb. Why is this not being fixed? It's kind of ridiculous... Here's Deepin's native dark theme:

![deepin_dark](https://user-images.githubusercontent.com/11800714/32406613-d93a1236-c17b-11e7-862b-ab068aa71948.png)

**Comment by jeffsauer** [comment]
still a problem with Deepin 15.5 release

**Comment by Xeers** [comment]
The problem remains unresolved [Arch-Linux]

**Comment by oberon2007** [comment]
Yeah, not only the file-manager, none of the native Deepin applications seem to accept gtk-theming.
Especially with the nice alternative dark theme included in deepin-gtk-theme, this seems to make little sense ... :stuck_out_tongue_closed_eyes:

**Comment by Az4zeel** [comment]
Use the decent native dark theme instead as a workaround to prevent retina burning :sweat_smile:

**Comment by dylanh724** [comment]
@Az4zeel What do you mean?

And also .... how is this not fixed yet? O_o

**Comment by ForeverZer0** [comment]
Seriously, over 2 years since this particular thread raised the issue (the issue existed well before that), and still is not fixed. This is getting to the point of ridiculousness, as this can absolutely ruin the overall look of an otherwise perfectly customized system. I like a dark theme, but the built-in dark theme is a bit too dark for my tastes and what the rest of the system uses, so it still sticks out and ruins the look.

**Comment by BLumia** [comment]
FYI make DTK apps themeable is planned but the priority is not high, we got limited resources working on these features. There will be more news about it in the upcomming future, please keep an eye on this :)

**Comment by SyntaxxxErr0r** [comment]
Hey Boys, and Girls... Sooo like many of you, i too have had this issue with this AMAZINGLY AWESOMELY COOL LOOKING THEME... like for reals. <3 this theme, this is my Kool-aide... it is generally common good advice to any one, do not mess with a mans kool-aide... Unless you're sure its gonna make it better, and not worse. :)

There is one quickish solution to this issue. and in fact it can actually be used on all themes with only a slight bit of tweaking to the CLI. 

For example.. if you have your copy of the theme  in your /usr/share/themes folder... 

`sed -i 's/border-color: none/border-color: transparent/g' "/usr/share/themes/deepin-dark/gtk-3.0/gtk.css"`

`sed -i 's/border-color: none/border-color: transparent/g' "/usr/share/themes/deepin-dark/gtk-3.0/gtk-dark.css"`

similiarly if you have it in your $HOME/.themes

`sed -i 's/border-color: none/border-color: transparent/g' "~/.themes/deepin-dark/gtk-3.0/gtk.css"`

`sed -i 's/border-color: none/border-color: transparent/g' "~/.themes/deepin-dark/gtk-3.0/gtk-dark.css"`

and if i am not mistaken there is also the bug in the same theme of a different variation.. where it is simply just a blank color... if memory serves me correctly i believe the syntax(x.x) (LOL) that i used was ...

`sed -i 's/border-color: \n/border-color: transparent\n/g' "/usr/share/themes/deepin-dark/gtk-3.0/gtk.css"`

`sed -i 's/border-color: \n/border-color: transparent\n/g' "/usr/share/themes/deepin-dark/gtk-3.0/gtk-dark.css"`

(This last suggestion may ben syntaxxxically err0roneous, and therefore i would appreciate any tips on the corrected command line, if any one has it.. as my moniker suggests... Syntaxxx Err0rs are common place for me. Typonese is my Native language. )
Cheers !! 

And BTW... did i happen to mention that I absolutely LOVE this fricken theme!!

**Comment by rjstechnet** [comment]
Hello,

Just curious is this will ever be fixed?
I use the Deepin desktop exclusively and have for years. Still have issues with not being able to theme the underlying deepin file manager etc....
If there is no intention of allowing to theme Deepin, It would be appreciated if the developers would say so.

Thank you!

**Comment by sdlins** [comment]
Deeping is nice, but it is buggy. Some bugs (like this) are very annoying and make our work harder. I changed to Mint about one year ago (just waiting new Ubuntu 20.04). That was the second and definitive time I gave out of Deepin. Never installed again.

However, I desire success to this promissor Linux Distribution.

**Comment by BLumia** [comment]
Oh, in case anyone would like to know, in next major release of DDE (v20), all Deepin app will get it's native dark theme support :)

**Comment by wishbon** [comment]
![DeepinScreenshot_Seleccionar área_20200427214757](https://user-images.githubusercontent.com/4439427/80437954-ff898880-88d0-11ea-8cf3-ef6724ae2e19.png)
